packages/aws-rfdk/docs/upgrade/upgrading-0.37.md

```diff
@@ -0,0 +1,41 @@
+# Upgrading to RFDK v0.37.x or Newer
```

Comment:
Great job with this upgrade doc. Do you mind adding a link to this from `packages/aws-rfdk/docs/upgrade/index.md`?

Reply:
Looks good! resolved